Shopify/cli
# use camelCase for hash keys in logs

### WHY are these changes introduced?

Fixes https://github.com/Shopify/shopify-functions/issues/271

### WHAT is this pull request doing?
Uses [camelcase-keys ](https://www.npmjs.com/package/camelcase-keys)to convert snake_case keys from appLogs.
I chose this because I saw it was already part of our dependencies https://github.com/Shopify/cli/blob/266908ce38dfef4cd28cb7a0eb38a7b2e8e6b4d5/pnpm-lock.yaml#L8361
... although I think I've just created another copy of it? 🤔 

### How to test your changes?
Generate an app log, click through to ensure that all keys are now in camelCase.
Run replay and see that:
- new runs look fine
- old runs now have an "undefined" timestamp, since they are missing the new `logTimestamp` (they have the old `log_timestamp`)
  - is this ok? Should we try to parse both old and new here? or is this break ok?

### Post-release steps



### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
